integration_tests/setup_test.go

```diff
@@ -176,21 +178,22 @@ func (s *IntegrationTestSuite) initNodesWithMnemonics(mnemonics ...string) {
 
 	//initialize a genesis file for the first validator
 	val0ConfigDir := s.chain.validators[0].configDir()
-	for _, val := range s.chain.validators {
+	for i, val := range s.chain.validators {
+		// Fund the first validator with some funds to be used by auction module integration tests
+		balanceStr := initBalanceStr
+		if i == 0 {
+			balanceStr += ",100000000000gravity0x3506424f91fd33084466f402d5d97f05f8e3b4af"
```

Comment:
I don't think we should initialize a validator with gravity funds that haven't been bridged through the contract in the test.